### DISCUSSION TITLE
macOS - Check /Library/ for server settings if none in ~/Library/

### DISCUSSION THREAD

**Discussion by e-lazer** [rustdesk/rustdesk#2683] [General]
I would like to deploy rustdesk via mdm on macOS, but I dont want to add files to the user library and/or user template. It would be nice if rustdesk would check in /Library/Preferences/com.carriez.RustDesk or /Library/Managed Preferences/com.carriez.RustDesk for example if no .toml exists in the user library. So you can push settings system wide.

**Comment by ghost** [comment]
I'm surprised that they do it per user rather than machine-wide.  Bad decision in my opinion. 

+1 for machine-wide settings.

**Comment by JonoScott-Stuff** [comment]
I'm suprised about this as well, if your going to to deploy it to systems where there are multiple users,  eg, family device etc,  you don't want to have to set up the settings for each user.

At least check to see if the configuration is in a machine wide folder first before defaulting to the users folder.
